src/main/features/dashboard/views/macro/claimStatus/helpWithFees.njk

```diff
@@ -272,9 +272,9 @@
   {{ t("We've reviewed your Help with Fees application") }}
 {% endmacro %}
 
-{% macro HWF_Intrest_Recalculate() %}
+{# {% macro HWF_Intrest_Recalculate() %}
```

Comment:
If this code is no longer required (removed as part of your story), please remove it instead of commenting it.